src/style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.searchbar_container {
+  padding: 0;
+  margin-top: 20px;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchbar,
+.searchbar::placeholder {
+  height: 100%;
+  width: 100%;
+  border: none;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  outline: none;
+}
```

Comment:
Most of these rules will be inherited, no need to apply all it to both class and placeholder